frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -520,7 +522,13 @@ end
 
 function ReaderHighlight:onReadSettings(config)
     self.view.highlight.saved_drawer = config:readSetting("highlight_drawer") or self.view.highlight.saved_drawer
-    self.view.highlight.disabled = config:readSetting("highlight_disabled") or false
+    if config:readSetting("highlight_disabled")~=nil then
+        self.view.highlight.disabled = config:readSetting("highlight_disabled")
+    elseif G_reader_settings:readSetting("highlight_disabled")~=nil then
+        self.view.highlight.disabled = G_reader_settings:readSetting("highlight_disabled")
+    else
+        self.view.highlight.disabled = false
+    end
```

Comment:
OOPS, i made a typo in my code snippet (`hightlight`). Luckily the automated static analysis caught it. Could you fix it?